static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -111,6 +111,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
                   project={feedbackItem.project}
                   size={12}
                   title={feedbackItem.project.slug}
+                  style={{boxShadow: 'none'}}
```

Comment:
this isn't really we have a pattern of doing, i think a styled ProjectAvatar with a selector to add this style to the element? is it overriding?

Reply:
it's not overriding because `ProjectAvatar` calls `PlatformList` which is what actually renders the project icon, but the styles aren't ever passed into `PlatformList`

Reply:
try something like this instead

```ts
const StyledProjectAvatar = styled(ProjectAvatar)`
  && img {
    box-shadow: none;
  }
`;
```